modules/data.atmosphere/R/met.process.R

```diff
@@ -93,7 +99,7 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
     }else{
 
       args <- list(outfolder, start_date, end_date)
-      if(met %in% "CRUNCEP") {
+      if((met %in% "CRUNCEP") | (met %in% "GFDL")) {
```

Comment:
I think this is fine for now, but moving forward we should add a tag to the registry for this type of download function rather than adding more special cases.